### ISSUE TITLE
List shouldn't default to LTR

### ISSUE THREAD

**Issue by mbrevda** [bvaughn/react-virtualized#457]
Pursuant to https://github.com/bvaughn/react-virtualized/issues/450, `List` also defaults to LTR. This is unnecessary, as List - by definition - doesn't scroll horizontally and works as expected in an RTL environment. Having LTR set causes otherwise properly rendered rows to render "backwards". 

New Property Proposal
---
Seeing as List simply provides a Grid, it might be necessary to pass in a prop so that Grid knows when `direction` settings are necessary.

To genericize this option, it might be called `direction` and have three options: `ltr`, `rtl`, or `auto`. 
 * `ltr` would act as it does now, forcing the style direction to `ltr` *if necessary* * and layout as it works today. 
 * `rtl` would force the layout to be flipped (i.e. #454), and would set the style *if necessary* *. 
 * `auto` would make an educated guess based on, say, `document.querySelector('html').dir`.

`direction` should probably default to `ltr` for now. Theoretically, once #454 is done, `auto` should "just work".

<br><br><br>
---
\* *once #454 is done, setting the direction shouldn't be necessary as the element should inherit from `html[dir]`*

**Comment by bvaughn** [comment]
This issue kind of redundant. #454 is sufficient.

For now, feel free to pass mode="rtl" to List if you want to use it a such. Once #454 is done, the default values can be changed.

**Comment by bvaughn** [comment]
Didn't mean to sound short or anything. Just didn't want another issue that so closely overlapped with ab existing one. Feel free to update #454 with those proposal.

(I don't think I will query the document for direction as to propose. This attribute can be set/inherited from any ancestor after all. The mode prop will likely stay around.)

**Comment by mbrevda** [comment]
Thanks for the quick response! I have updated the other issue with a note to remove `direction` once that issue is resolved. Thanks!

p.s. communication is hard! 🤗

**Comment by bvaughn** [comment]
Thanks buddy! :smile:
